NuKeeper.Tests/RepositoryInspection/PackagesFileReaderTests.cs

```diff
@@ -69,8 +70,14 @@ public void TwoPackagesShouldBeRead()
 
             Assert.That(packages, Is.Not.Null);
             Assert.That(packages.Count, Is.EqualTo(2));
+
+            PackageAssert.IsPopulated(packages[0]);
             Assert.That(packages[0].Id, Is.EqualTo("foo"));
+            Assert.That(packages[0].Version, Is.EqualTo(new NuGetVersion("1.2.3.4")));
+
+            PackageAssert.IsPopulated(packages[1]);
             Assert.That(packages[1].Id, Is.EqualTo("bar"));
+            Assert.That(packages[1].Version, Is.EqualTo(new NuGetVersion("2.3.4.5")));
```

Comment:
This is a lot of asserts for a single test. 16 of them if we unpack `PackageAssert.IsPopulated`